src/components/FeatureViews/Accounts/TabComponents/CourseSessionStatus.js

```diff
@@ -495,7 +495,7 @@ const CourseSessionStatus = () => {
                 <DialogContent>
                     <DialogContentText>
                         {`
-                        You are currently registering ${registeringParent && registeringParent.user.name}. If you wish to continue to add sessions, you will
+                        You are currently registering ${registeringParent.user && registeringParent.user.name}. If you wish to continue to add sessions, you will
```

Comment:
This might break if `registeringParent` is undefined/null, not sure if that is undefined/null ever. You would need all 3 checks here if so.